es-spatial-plugin/project.clj

```diff
@@ -86,18 +92,31 @@
                        "shell" "zip" "-j" ~plugin-zip-name ~uberjar-name]
 
             ;; Packages and installs the plugin into the local elastic search vm
-            "install-local" ["do"
-                             "package,"
-                             "shell" "../../cmr-vms/elastic_local/install_plugin.sh" ~plugin-zip-name "spatialsearch-plugin,"
-                             "clean"]
+            "install-local" [
+              "do"
+              "package,"
+              "shell"
+                "../../cmr-vms/elastic_local/install_plugin.sh"
+                ~plugin-zip-name
+                "spatialsearch-plugin,"
+              "clean"]
 
-            "install-aws" ["do"
-                           "package,"
-                           ;; IP address is hard coded for now
-                           "shell" "../cmr-vms/elastic_aws/install_plugin.sh" "54.193.23.62" ~plugin-zip-name "spatialsearch-plugin"]
-            "install-workload" ["do"
-                                "package,"
-                                "shell" "install_plugin_into_workload.sh" ~plugin-zip-name "spatialsearch-plugin"]
+            "install-aws" [
+              "do"
+              "package,"
+              ;; IP address is hard coded for now
+              "shell"
+                "../cmr-vms/elastic_aws/install_plugin.sh"
+                "54.193.23.62"
```

Comment:
We should look into this - I don't remember elastic_aws at all - I don't think it was related to moving CMR to the cloud. Was this part of prototyping a really long time ago (i.e. prior to the CMR)? We should ideally get rid of this.

Reply:
Yeah, looks like the last change in `crm-vms/elastic_aws` was in 2014-01-29. The IP doesn't respond to an HTTP `GET` ... anyway, seeing how our infrastructure has completely changed (probably several times), I'm 👍 for removal ...